src/gluonts/model/rotbaum/_preprocess.py

```diff
@@ -260,6 +260,8 @@ def __init__(
         use_feat_static_cat=False,
         use_feat_dynamic_real=False,
         use_feat_dynamic_cat=False,
+        static_cardinality: Optional[List] = None,
+        one_hot_encode: bool = True,  # should improve performance but will slow down model
```

Comment:
Can we use "accuracy" instead of "performance"? I think that term is less ambiguous.